.travis.yml

```diff
@@ -7,6 +7,7 @@ cache:
 sudo: false
 env:
   global:
+  - MAVEN_WRAPPER_ALWAYS_DOWNLOAD=true
```

Comment:
As discussed offline until the maven wrapper library supports checksum for the downloaded packages I was erring on the side of caution here. 

Looking at this a little deeper it seems that this will not be cached by Travis regardless of the setting. The reason is that the wrapper does not download the maven package as a normal maven dependency and it is stored by default outside the cache directory (e.g in ~/.m2/maven instead of ~/.m2/repository); for details see:

https://github.com/rimerosolutions/maven-wrapper/wiki/Maven-Goals

Given this is the case I've removed the setting.